test/python/circuit/test_initializer.py

```diff
@@ -61,6 +62,21 @@ def test_deterministic_state(self):
             fidelity, self._desired_fidelity,
             "Initializer has low fidelity {:.2g}.".format(fidelity))
 
+    def test_statevector(self):
```

Comment:
Wouldn't it be sufficient to check that the `params` in `Initialize` are actually the same as `Statevector.data`? Since `initialize` itself is already tested I don't think we have to test the synthesis here.

Reply:
Sure. Fixed in 75f85927b

Reply:
sure. simpler test in 75f8592